modules/openapi-generator/src/main/java/org/openapitools/codegen/languages/DartClientCodegen.java

```diff
@@ -414,7 +416,7 @@ public String toDefaultValue(Schema schema) {
             }
             return schema.getDefault().toString();
         } else {
-            return "null";
+            return null;
```

Comment:
This fixes not setting `null` value on properties that don't have default value. Dart is evolving to non-null by default language, so this will be important in the next releases of the language.